apps/cowswap-frontend/src/modules/account/containers/OrdersPanel/index.tsx

```diff
@@ -45,7 +45,7 @@ const SideBar = styled.div`
     height: 100%;
     max-width: 100%;
     border-radius: ${({ theme }) => (theme.isInjectedWidgetMode ? '24px' : '0')};
-    z-index: 10000;
+    z-index: 10;
   }
```

Comment:
Alright! @anxolin I've digged more deep into the issue my finding and reasoning for the changes that I've made.

- So the main here is that in mobile view the `Account` modal have insanely high z-index values which casuing every other open modal to be stacked behide the `Account` Modal, Now I've adjusted this to fit with the rest of the modal in this page and not to conflict with them.